pkg/observability/config.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Google LLC
```

Comment:
We might want to consider making the package in the main server public instead of duplicating this.

Reply:
Yeah, going to do in a followup PR. Added a TODO